### PR TITLE
Run nightly tests on dispatch

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: .github/workflows/scripts/script.sh [hunk 1]
```diff
@@ -164,7 +164,8 @@ if [ -f $FUNC_TEST_SCRIPT ]; then
   source $FUNC_TEST_SCRIPT
 else
 
-    if [[ "$GITHUB_EVENT_NAME" == "schedule" ]]; then
+    echo $GITHUB_EVENT_NAME
+    if [[ "$GITHUB_EVENT_NAME" == "schedule" || "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]]; then
```
[the remark this answers is not in the document]
Comment (bmbouter) [reply .github/workflows/scripts/script.sh:168]: How should we test this?

[the remark this answers is not in the document]
Comment (fao89) [reply .github/workflows/scripts/script.sh:168]: merging and triggering it
I put an echo to get the name if this is one is not the right one

[the remark this answers is not in the document]
Comment (bmbouter) [reply .github/workflows/scripts/script.sh:168]: Oh I missed this was on pulp_ansible (I thought it was plugin_template). Perfect!

**Review by bmbouter — APPROVED** [review]

**Comment by bmbouter** [comment]
Let's see https://github.com/pulp/pulp_ansible/actions/runs/1796608160

**Pull request merged**